src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -92,6 +109,11 @@ private List<MembershipResult> createMemberships(List<Group> membershipGroupings
             membershipResult.setDescription(grouping.getDescription());
             membershipResult.setPath(grouping.getGroupPath());
             membershipResult.setName(nameGroupingPath(grouping.getGroupPath()));
+            GroupAttributeResults groupingAttributeResults =
+                    grouperService.groupAttributeResults(CURATED, grouping.getGroupPath());
+            if (groupingAttributeResults.getResultMessage().equals(", Found 1 results.")) {
+                membershipResult.setName(grouping.getGroupPath());
+            }
```

Comment:
Does checking `groupingAttributeResults.getResultCode().equals("SUCCESS")` here work as well or is the resultCode not sufficient enough compared to checking the message?